### PR TITLE
User guide: troubleshooting: describe the issue with forced deletion

### PR DESCRIPTION
**Pull request by dtantsur**
This PR starts a troubleshooting FAQ for BMO. The first two entries
concern stuck cleaning and conflict after a forced deletion.

Signed-off-by: Dmitry Tantsur <dtantsur@protonmail.com>

### CODE REVIEW

File: docs/user-guide/src/bmo/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+host will be deleted. If you do care about cleaning, you need to figure out why
+it does not finish.
+
+## MAC address conflict on registration
+
+If you force deletion of a host after registration, BareMetal Operator will not
+be able to delete the corresponding record from Ironic. If you try to enroll
+the same host again, you will see the following error:
+
+`Normal  RegistrationError  4m36s  metal3-baremetal-controller  MAC address 11:22:33:44:55:66 conflicts with existing node namespace~name`
```
Comment (tuminoid) [inline docs/user-guide/src/bmo/troubleshooting.md:25]: Use three ticks instead of one as it is a paragraph of its own and not embedded in sentence.

**Review by tuminoid — COMMENTED** [review]
Formatting suggestion.

File: docs/user-guide/src/bmo/troubleshooting.md (hunk 1 above)
Comment (dtantsur) [reply docs/user-guide/src/bmo/troubleshooting.md:25]: The reason I haven't done it is:

docs/user-guide/src/bmo/troubleshooting.md:25 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]

Comment (tuminoid) [reply docs/user-guide/src/bmo/troubleshooting.md:25]: It complains if you have  ` '''console` (replace '''' with actual ticks) ?

Comment (tuminoid) [reply docs/user-guide/src/bmo/troubleshooting.md:25]: Maybe just '''text block then, if you feel console is wrong? I think we have quite a lot of plain output marked as console?

**Comment by dtantsur** [comment]
@tuminoid not sure I love the horizontal scrolling but it works. PTAL.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
Thanks!
/approve

**Pull request merged**